modules/EnsEMBL/Web/Component/Gene/ComparaOrthologs.pm

```diff
@@ -92,30 +97,23 @@ sub content {
     next if $species eq $hub->species; ## Ignore current species
     my $label = $is_pan ? $pan_lookup->{$prod_name}{'display_name'} : $species_defs->species_label($species);
 
-    ## Should we be showing this orthologue on this pagpe by default?
-    my $strain_group  = $species_defs->get_config($species, 'STRAIN_GROUP');
-    my $related_taxon = $species_defs->get_config($species, 'RELATED_TAXON');
-    if ($hub->action =~ /^Strain_/) {
-      unless (($strain_group && $strain_group eq $this_group) || ($related_taxon && $related_taxon eq $species_defs->RELATED_TAXON)) {
```

Comment:
Issue ENSCOMPARASW-8511 is related to this requirement for either the `STRAIN_GROUP` or `RELATED_TAXON` to match in strain views.

This could be circumvented by checking instead for whether the production name is in the `orthoset_prod_names` set for the given strain gene-tree collection.